stock_quant_manual_assign/wizard/assign_manual_quants_view.xml

```diff
@@ -11,15 +11,11 @@
                             <tree editable="top" delete="0" create="0">
                                 <field name="quant" />
                                 <field name="location_id" />
+                                <field name="lot_id" />
```

Comment:
This should have group, so that you make it visible only for those in "group_stock_production_lot"